ferrocene/doc/internal-procedures/src/subtree-pulls.rst

```diff
@@ -83,8 +83,11 @@ You can also pass the name of a repository to only pull that specific repo::
    ferrocene/tools/pull-subtrees/pull.py <SUBTREE_REPO>
 
 Note that `<SUBTREE_REPO>` must be the repository of the subtree and not its path.
+An example is ``ferrocene/tools/pull-subtrees/pull.py rust-lang/fls``,
```

Comment:
Could we break this into a separate paragraph? As it stands it appears kind of mooshed in with the note:

![image](https://github.com/user-attachments/assets/914d14af-7c6f-4207-b069-252f70b0863b)